kessel-sdk/src/main/java/org/project_kessel/api/rbac/v2/FetchWorkspace.java

```diff
@@ -0,0 +1,67 @@
+package org.project_kessel.api.rbac.v2;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.project_kessel.api.auth.AuthRequest;
+import org.project_kessel.api.auth.OAuth2Exception;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class FetchWorkspace {
+
+    private static final String WORKSPACE_ENDPOINT = "/api/rbac/v2/workspaces/";
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    public static Workspace fetchRootWorkspace(String rbacBaseEndpoint, String orgId, FetchWorkspaceOptions options)
```

Comment:
About the `FetchWorkspaceOptions` -  I'm not sure if this is a Java pattern, but i don't recall it being one.

With this number of parameters, we could just overload the `fetchRootWorkspace` method and call it with a different number of params and/or using the [Optional class](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html).
e.g.

```java

public static Workspace fetchDefaultWorkspace(String rbacBaseEndpoint, String orgId) // throws... {
    fetchDefaultWorkspace(rbacBaseEndpoint, orgId, null, null); // or Optional.empty(), Optional.empty()
}

public static Workspace fetchDefaultWorkspace(String rbacBaseEndpoint, String orgId, AuthRequest authRequest) // throws {
    fetchDefaultWorkspace(rbacBaseEndpoint, orgId, authRequest, null);
}
// etc

```

Reply:
Went ahead with just nulling things and method overloading. No more options object to instantiate